src/plugins/vis_augmenter/public/vega/helpers.ts

```diff
@@ -21,6 +21,8 @@ import {
   HOVER_PARAM,
 } from '../';
 
+const showEventsFlyoutAction = 'show-events-flyout';
```

Comment:
Do you want to take on the view events flyout action, or just have the framework set up that I can build on top of? For example I already have the defined open flyout action in my draft PR here: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3415/files#diff-e5581c4c9ed88d345c2fffa25e66f906e98b05d699e4c5614a87748e31996d4d